department-of-veterans-affairs/abd-vro
# svc-bip-api: General cleanup

## What was the problem?

After all the rework done on the `svc-bip-api` there was a lot of unused code and some disorganization.

## How does this fix it?[^1]

Remove unused classes and general clean up.

## How to test this PR
Successful CI run.

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
